eventlogger/src/test/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+spring.datasource.url=${DB_URL}
```

Comment:
Does this work without this file? The modules that aren't main ones have names like application.audit.properties and don't need to specify things like `spring.datasource.username` that are specified elsewhere.

Reply:
I just got a bunch or errors without a lot of those properties because I was dependent on common. I might actually move this code into common if common needs to do any logging, otherwise, I get a circular dependency.